README.md

```diff
@@ -164,17 +164,17 @@ In this next step we'll run `model-archiver` and tell it our model's prefix is `
 model-archiver --model-name squeezenet_v1.1 --model-path squeezenet --handler mxnet_vision_service:handle
 ```
 
-This will package all the model artifacts files in `squeezenet` directory and output `squeezenet_v1.1.mar` in the current working directory. This `.mar` file is all you need to run MMS, serving inference requests for a simple image recognition API. Go back to the Serve a Model tutorial above and try to run this model that you just exported!
+This will package all the model artifacts files in `squeezenet` directory and output `squeezenet_v1.1.mar` in the current working directory. This `.mar` file is all you need to run MMS, serving inference requests for a simple image recognition API. Go back to the Serve a Model tutorial above and try to run this model archive that you just created!
 
-To learn more about exporting, check out [Model archiver documentation](../model-archiver/docs/README.md)
+To learn more about `model-archiver`, check out [Model archiver documentation](model-archiver/README.md)
 
 ## Recommended production deployments
 
-* MMS doesn't provide authentication. You have to your own authentication proxy in front of MMS. 
-* MMS doesn't provide throttling, it's vulnerable to DDoS attack. It's recommended to running MMS behind a firewall.
-* MMS only allows localhost access by default, see [Network configuration](administration.md#network) for detail.
-* SSL is not enabled by default, see [Enable SSL](administration.md#enable_ssl) for detail.
-* MMS use a config.properties file to configure MMS's behavior, see [Manage MMS](administration.md) page for detail of how to configure MMS.
+* MMS doesnt provide authentication. You have to your own authentication proxy in front of MMS. 
```

Comment:
Doesn't

Reply:
done, somehow pycharm this doesn't is typo and autofixed it.